tests/test_utils.py

```diff
@@ -225,48 +241,55 @@ def get_dataset(
                 elif type_ in ["DATETIME", "TIMESTAMP"]:
                     df[col] = pd.to_datetime(df[col])
 
-        tablename = "test_data_" + "".join(
-            [random.choice(string.ascii_letters + string.digits) for n in range(8)]
-        )
-        df.to_sql(name=tablename, con=conn, index=False, dtype=sql_dtypes)
-
-        # Build a SqlAlchemyDataset using that database
-        return SqlAlchemyDataset(
-            tablename, engine=conn, profiler=profiler, caching=caching
-        )
-
-    elif dataset_type == "mysql":
-        from sqlalchemy import create_engine
-
-        engine = create_engine("mysql://root@localhost/test_ci")
-        conn = engine.connect()
-
-        sql_dtypes = {}
-        if (
-            schemas
-            and "mysql" in schemas
-            and isinstance(engine.dialect, mysqltypes.dialect)
-        ):
-            schema = schemas["mysql"]
-            sql_dtypes = {col: MYSQL_TYPES[dtype] for (col, dtype) in schema.items()}
-            for col in schema:
-                type_ = schema[col]
-                if type_ in ["INTEGER", "SMALLINT", "BIGINT"]:
-                    df[col] = pd.to_numeric(df[col], downcast="signed")
-                elif type_ in ["FLOAT", "DOUBLE", "DOUBLE_PRECISION"]:
-                    df[col] = pd.to_numeric(df[col])
-                elif type_ in ["DATETIME", "TIMESTAMP"]:
-                    df[col] = pd.to_datetime(df[col])
-
-        tablename = "test_data_" + "".join(
-            [random.choice(string.ascii_letters + string.digits) for n in range(8)]
+        if table_name is None:
+            table_name = "test_data_" + "".join(
+                [random.choice(string.ascii_letters + string.digits) for _ in range(8)]
+            )
+        df.to_sql(
+            name=table_name,
+            con=conn,
+            index=False,
+            dtype=sql_dtypes,
+            if_exists="replace",
         )
-        df.to_sql(name=tablename, con=conn, index=False, dtype=sql_dtypes)
 
         # Build a SqlAlchemyDataset using that database
         return SqlAlchemyDataset(
-            tablename, engine=conn, profiler=profiler, caching=caching
+            table_name, engine=conn, profiler=profiler, caching=caching
         )
+    #
+    # elif dataset_type == "mysql":
```

Comment:
I'd like to understand your thinking behind commenting this out.

Have you had a chance to peruse #1497 ? There's some good compare-and-contrast there.

Also:
* https://github.com/EarnestResearch/great_expectations/pull/114
* https://github.com/EarnestResearch/great_expectations/pull/9

Reply:
I wanted to clarify true status. This PR adds new features for testing, and I didn't want to leave an incompletely-implemented and unused codepath active until it is also brought up to speed.

Reply:
Rather than comment it out, I'd advocate for adding a mysql docker image in `assets/docker` parallel to postgresql.

Reply:
Done -- I've added a container, instructions for locally running, updated our travis script, updated the test runner, etc.